css/properties.json

```diff
@@ -5945,6 +5945,21 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "overflow-anchor": {
+    "syntax": "auto | none",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Box Model"
+    ],
+    "initial": "auto",
+    "appliesto": "allElements",
+    "computed": "asSpecified",
+    "order": "perGrammar",
+    "status": "standard"
```

Comment:
Since this is an Editor's Draft and is only implemented by one browser engine AFAIK (https://caniuse.com/#feat=css-overflow-anchor) I think experimental might be more appropriate here? Our guidance on this should be clearer though.

Reply:
You're right. Fixed.